ironfish/src/rpc/routes/chain/followChainStream.ts

```diff
@@ -86,36 +86,34 @@ routes.register<typeof FollowChainStreamRequestSchema, FollowChainStreamResponse
 
     const send = (block: Block, type: 'connected' | 'disconnected' | 'fork') => {
       const transactions = block.transactions.map((transaction) => {
-        return transaction.withReference(() => {
-          return {
-            ...(request.data?.serialized
-              ? { serialized: transaction.serialize().toString('hex') }
-              : {}),
-            hash: BlockHashSerdeInstance.serialize(transaction.hash()),
-            size: getTransactionSize(transaction),
-            fee: Number(transaction.fee()),
-            expiration: transaction.expiration(),
-            notes: transaction.notes.map((note) => ({
-              commitment: note.hash().toString('hex'),
-            })),
-            spends: transaction.spends.map((spend) => ({
-              nullifier: spend.nullifier.toString('hex'),
-              commitment: spend.commitment.toString('hex'),
-              size: spend.size,
-            })),
-            mints: transaction.mints.map((mint) => ({
-              id: mint.asset.id().toString('hex'),
-              metadata: BufferUtils.toHuman(mint.asset.metadata()),
-              name: BufferUtils.toHuman(mint.asset.name()),
-              creator: mint.asset.creator().toString('hex'),
-              value: mint.value.toString(),
-            })),
-            burns: transaction.burns.map((burn) => ({
-              id: burn.assetId.toString('hex'),
-              value: burn.value.toString(),
-            })),
-          }
-        })
+        return {
```

Comment:
Nit: I think the `return` is not needed here:

```
const transactions = block.transactions.map((transaction) => {
  return { ... }
}
```

can become:


```
const transactions = block.transactions.map((transaction) => {
  ...
}
```

saving 1 more level of indentation.

Reply:
Ah yeah, I'll have to wrap it in parens, but otherwise that'll work. I'll update it